src/main/graphql/GetConsignmentStatus.graphql

```diff
@@ -0,0 +1,10 @@
+query getConsignmentStatus($consignmentId: UUID!) {
+    getConsignment(consignmentid: $consignmentId) {
+        series {
+            code
```

Comment:
Is the series field necessary? If it is, perhaps the name of this query should be different to make it clear when you'd want this combination of fields?

Reply:
The series isn't necessary, I think I just had it in based on the examples we were talking about when we were figuring out the structure of the currentStatus stuff. I can remove it 👍